rdagent/utils/env.py

```diff
@@ -883,7 +883,11 @@ def _run(
 
         if self.conf.extra_volumes is not None:
             for lp, rp in self.conf.extra_volumes.items():
-                volumes[lp] = rp if isinstance(rp, dict) else {"bind": rp, "mode": self.conf.extra_volume_mode}
+                volumes[lp] = (
```

Comment:
I think you can specify the permission outside, instead of hardcode in the infra